client/components/pages/login/LoginForm.vue

```diff
@@ -0,0 +1,115 @@
+<script lang="ts" setup>
+import { object, string } from 'yup'
+import { useForm } from 'vee-validate'
+
+const { t, m } = useI18n()
+const { login } = useAuth()
+
+const usernamePlaceholder = getRandomDeveloperUsername()
+const message = defineModel<{
+  content: string | null
+  severity: 'success' | 'info' | 'warn' | 'error' | undefined
+}>('message', {
+  default: reactive({
+    content: null,
+    severity: undefined,
+  }),
```

Comment:
```suggestion
  default: () => ({
    content: null,
    severity: undefined,
  }),
```

(or just a simple object maybe)
The object is static anyway, not reactive